website/docs/docs/core/connect-data-platform/about-core-connections.md

```diff
@@ -39,20 +39,20 @@ The following table lists the features available for adapters:
 
 | Adapter | Catalog | Source freshness |
 |---------|---------|------------------|
-| dbt default configuration | full | `loaded_at` field |
-| `dbt-bigquery` | partial and full | metadata-based and `loaded_at` field |
-| `dbt-databricks` | full | metadata-based and `loaded_at` field |
-| `dbt-postgres` | partial and full | `loaded_at` field |
-| `dbt-redshift` | partial and full | metadata-based and `loaded_at` field |
-| `dbt-snowflake` | partial and full | metadata-based and `loaded_at` field |
-| `dbt-spark` | full | `loaded_at` field |
+| dbt default configuration | full | `loaded_at_field` |
+| `dbt-bigquery` | partial and full | metadata-based and `loaded_at_field` |
+| `dbt-databricks` | full | metadata-based and `loaded_at_field` |
+| `dbt-postgres` | partial and full | `loaded_at_field` |
+| `dbt-redshift` | partial and full | metadata-based and `loaded_at_field` |
+| `dbt-snowflake` | partial and full | metadata-based and `loaded_at_field` |
+| `dbt-spark` | full | `loaded_at_field` |
 
 
 ### Catalog 
 
 For adapters that support it, you can partially build the catalog. This allows the catalog to be built for only a select number of models via `dbt docs generate --select ...`. For adapters that don't support partial catalog generation, you must run `dbt docs generate` to build the full catalog.
 
 ### Source freshness
-You can measure source freshness using the warehouse metadata tables on supported adapters. This allows for calculating source freshness without using the `loaded_at` field and without querying the table directly. This is faster and more flexible (though it might sometimes be inaccurate, depending on how the warehouse tracks altered tables). You can override this with the `loaded_at` field in the model config. If the adapter doesn't support this, you can still use the `loaded_at` field.
+You can measure source freshness using the warehouse metadata tables on supported adapters. This allows for calculating source freshness without using the `loaded_at_field` and without querying the table directly. This is faster and more flexible (though it might sometimes be inaccurate, depending on how the warehouse tracks altered tables). You can override this with the `loaded_at_field` in the model config. If the adapter doesn't support this, you can still use the `loaded_at_field`.
```

Comment:
non blocker but wondering if we should link to it here? or wrong link?

```suggestion
You can measure source freshness using the warehouse metadata tables on supported adapters. This allows for calculating source freshness without using the [`loaded_at_field`]/reference/resource-properties/freshness#loaded_at_field) and without querying the table directly. This is faster and more flexible (though it might sometimes be inaccurate, depending on how the warehouse tracks altered tables). You can override this with the `loaded_at_field` in the model config. If the adapter doesn't support this, you can still use the `loaded_at_field`.
```